.github/workflows/build-test-radosgw.yml

```diff
@@ -55,23 +54,55 @@ jobs:
           sudo apt-get update
           sudo apt-get install -y s3cmd
 
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.8
+
+      - name: Run Integration tests
```

Comment:
Optional: Can we run the integration and smoke tests in parallel?

Reply:
In theory yes. In practice, distributing the rather large build artifacts between multiple worker instances takes so much time that any gain we would make is lost tenfold, since integration and smoke tests complete in under a minute if run sequentially. I'll keep it in the back of my mind in case we extend the test suites.